templates/taxi/manufacturer_list.html

```diff
@@ -1,7 +1,7 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
+    <h1>Manufacturer List <a href="{% url 'taxi:manufacturer-create' %}">+</a>
     </h1>
 
```

Comment:
Code Style Issue: The indentation in this HTML file should be 2 spaces according to the checklist. Please adjust the indentation throughout the file.